packages/rubrik/data_stream/virtualmachines/elasticsearch/ingest_pipeline/default.yml

```
@@ -29,9 +29,21 @@ processors:
       field: json.name
       target_field: rubrik.virtualmachine.name
       ignore_missing: true
+  - rename:
+      field: json.cluster.id
+      target_field: rubrik.virtualmachine.cluster.id
```

```suggestion
      target_field: rubrik.virtual_machine.cluster.id
```
For consistency with other data streams, I think we should name it like this.